### PR TITLE
Added --rewrite argument supporting rewrite rule

### PR DESCRIPTION
**Pull request by mangchiandjjoe**

### CODE REVIEW

File: start_esp/server-auto.conf.template [hunk 1]
```diff
@@ -31,8 +31,12 @@ service_management_config {
 % endif
 experimental {
   disable_log_status: true
-% if always_print_primitive_fields:
```
Comment (qiwzhang) [inline start_esp/server-auto.conf.template]: please sync to the latest

**Review by qiwzhang — CHANGES_REQUESTED** [review]

File: start_esp/start_esp.py [hunk 2]
```diff
@@ -490,6 +491,15 @@ def make_argparser():
     display this error. Setting size to 0 disables checking of client request
     body size.''')
 
+    parser.add_argument('--rewrite', action='append', help=
+    '''Internally redirect the request uri with a pair of pattern and
+    replacement. If the request uri matches perl compatible regular expression,
```
Comment (qiwzhang) [inline start_esp/start_esp.py]: please mention the pair is separated by space

Comment (qiwzhang) [reply start_esp/start_esp.py]: add 1, or 2 samples

File: start_esp/server-auto.conf.template (hunk 1 above)
Comment (mangchiandjjoe) [reply start_esp/server-auto.conf.template]: Done

File: start_esp/start_esp.py (hunk 2 above)
Comment (mangchiandjjoe) [reply start_esp/start_esp.py]: Updated comments

**Comment by mangchiandjjoe** [comment]
PTAL

**Review by qiwzhang — APPROVED** [review]

**Pull request merged**